pkg/parser/ast/ddl.go

```diff
@@ -830,6 +850,32 @@ func (n *IndexOption) Accept(v Visitor) (Node, bool) {
 		return v.Leave(newNode)
 	}
 	n = newNode.(*IndexOption)
+	if n.SplitOpt != nil {
+		for i, val := range n.SplitOpt.Lower {
```

Comment:
it better to let `SplitOption` implement `Accept`, so here and `SplitRegionStmt` won't repeat